src/actions/CryptoExchangeActions.js

```diff
@@ -164,6 +164,10 @@ export const shiftCryptoCurrency = () => async (dispatch: Dispatch, getState: Ge
             name = 'Changelly'
             supportEmail = 'support@changelly.com'
             break
+          case 'faast':
```

Comment:
@ahujsak The plugins already export this information from their `swapInfo`. There should be absolutely no reason for this information to exist in the GUI. We should take a look at fixing this before we add more plugins so it doesn't get worse.

Reply:
@swansontec One additional thing I noticed is that the Exchange settings page doesn't use pluginName. It looks like it's using a capitalized version of the plugin code name instead.